courses/docker_beginner_fr_zerofiltre.yaml

```diff
@@ -0,0 +1,17 @@
+id: null
+name: Docker 
+url: https://zerofiltre.tech/cours/30-déployez-votre-application-dans-tout-environnement-gr-ce-à-docker
+duration_hours: 5
+level: Beginner
+objectives: Apprendre les bases de Docker pour conteneuriser les applications.
+description: >
+  Docker garantit que les applications fonctionnent de manière identique dans tous les environnements, éliminant les problèmes de compatibilité et "ça marche sur ma machine 😅".
```

Comment:
il faut éviter les emojis, les <'> et les <"> dans les textes
tu peux corriger ? 
Avoid french character Please if you want to use <'> (apostrophe in french) please use < >(space) instead So if you want to write <l'initialisation> please use < l initiatilisation > instead

Reply:
Yes sure, je le corrige de suite